src/pages/login.tsx

```diff
@@ -23,177 +25,89 @@ export default function Home() {
         withCredentials: true,
       });
 
-
       console.log('Login successful:', loginResponse.data);
       const contentType = loginResponse.headers['content-type'];
       if (contentType && contentType.includes('text/html')) {
         setShowWarning(true);
         return;
-
-      }
-      else
-      {
-        window.location.href = '/'
+      } else {
+        window.location.href = '/';
       }
-
     } catch (error) {
-
       console.error('Login failed:', error);
       alert('Login failed. Please check your credentials and try again.');
```

Comment:
Kindly take a look at [react-hot-toast](https://react-hot-toast.com), integrate it for neat UI&UX.